Jenkinsfile

```diff
@@ -35,15 +35,15 @@ pipeline {
 
     stage('L0: Unit Tests GPU') {
       steps {
-        sh 'pytest -m unit'
+        sh 'pytest -m "unit and not skipduringci"'
```

Comment:
I am not really sure why we need that for...

The simplest way to skip a test during CI is *not use any of the existing markers* - it test is not "unit", "integration", "system" or "unclassified", it won't be executed during CI...

Reply:
I should all tests should be classified. However in the case of TTS, there was an overlap in testing tacotron2 since it was both an integration test and a CI test. I think it would be beneficial to have this flag, if in the future, we want to pick which test to run.

Reply:
ok, so let us leave it for now